errai-bus/src/main/java/org/jboss/errai/bus/server/servlet/CSRFTokenFilter.java

```diff
@@ -32,6 +32,7 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 
+import org.jboss.errai.common.server.FilterCacheUtil.CharResponseWrapper;
```

Comment:
I believe you don't need this import any longer.

Reply:
new code need it